.changeset/clean-ravens-melt.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/core-typings": minor
+---
+
+Using the `Livechat_Block_Unknown_Contacts`, `Livechat_Contact_Verification_App`, `Livechat_Request_Verification_On_First_Contact_Only` settings to handle the visitor to a verificator app if needed.
```

Comment:
Think this can be improved a bit, describing more on what was done and less on the setting ids involved to do it, wdyt?